samples/matter/light_switch/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ get_filename_component(MATTER_MODULE_ROOT $ENV{ZEPHYR_BASE}/../modules/lib/matte
 # Set Kconfig root files that will be processed as a first Kconfig for used child images.
 set(mcuboot_KCONFIG_ROOT ${MATTER_MODULE_ROOT}/config/nrfconnect/chip-module/Kconfig.mcuboot.root)
 set(multiprotocol_rpmsg_KCONFIG_ROOT ${MATTER_MODULE_ROOT}/config/nrfconnect/chip-module/Kconfig.multiprotocol_rpmsg.root)
+set(hci_rpmsg_KCONFIG_ROOT ${MATTER_MODULE_ROOT}/config/nrfconnect/chip-module/Kconfig.hci_rpmsg.root)
```

Comment:
Maybe we could consider setting network image kconfig root depending on the BOARD_NAME used, to not add both images configs and confuse user a little bit?

Reply:
On the other hand, we don't set mcuboot's Kconfig root only if the DFU is used. I'm not sure if adding a lot of conditionals to the project makes it easier to understand. I would rather try to write the application in a way that avoids "if platform is ..." stuff, but if the majority of us thinks differently, I will adapt :)

Reply:
Ok, fair point, it convinced me :)